### ISSUE TITLE
`querySelector` multi-selector fails when both parts refer to same attribute

### ISSUE THREAD

**Issue by Macil** [jsdom/jsdom#2403]
### Basic info:

- **Node.js version:** v10.11.0
- **jsdom version:** 12.2.0 (latest), also 11.12.0 (currently used by jest)

### Minimal reproduction case

```js
'use strict';
const jsdom = require("jsdom");
const { JSDOM } = jsdom;

const dom = new JSDOM(`
  <!doctype html>
  <html>
  <body>
    <div>
      <div tabindex="0">
      </div>
    </div>
  </body>
  </html>
`);

const el1 = dom.window.document.querySelector('div div [tabindex], [tabindex]');
console.log('el1', el1);

const el2 = dom.window.document.querySelector('div div [tabindexZ], [tabindex]');
console.log('el2', el2);
console.log('el1 === el2', el1 === el2);
```

Both of the querySelector calls should return the same element, but the first querySelector call incorrectly returns null.

**Comment by Zirro** [comment]
Thanks for the report!

@dperini, would you like to take a look at this?

**Comment by dperini** [comment]
Ok ! I will have a look at this issue.

**Comment by Macil** [comment]
This seems to be fixed by nwsapi version 2.1.0. I did find a new bug though, so I've linked it to this issue in case it's related.
